.github/workflows/issue.yml

```diff
@@ -0,0 +1,15 @@
+name: Labeling new issue
+on:
+  issues:
+    types: ['opened']
```

Comment:
Should this be added to every issue? I think that if a bug is properly defined, this label is only necessary for feature request

Reply:
Certain bugs have a higher priority than others. Every issue will need to be triaged to be prioritized.